CMakeLists.txt

```diff
@@ -382,7 +386,6 @@ endforeach()
 add_custom_target(demo DEPENDS ${DEMO_EXECS})
 
 # Tests
-if(NOT(ANDROID_BUILD OR AAR_BUILD))
```

Comment:
In order to test Tensorflow, TFLite and Hexagon adapters on Android we need to enable GTest programs build for Android as well.

Reply:
On a separate note, is it better to disable gtest by default? Maybe put it in a 'test' target and exclude it from 'all', since it adds quite some build time and the repos might not be accessible from some edge devices.

Reply:
What you think if we GNU toolchain to build

Reply:
I think we're already able to do that? E.g. I was able to compile dlr_tflite_test for imx8 with aarch64 crosscompiler designated in a toolchain.cmake.

Reply:
Great!. Lets add ARM Linux cross-compilation to `install.rst`